src/commands/deploy/nextjs/deploy.ts

```diff
@@ -391,7 +428,22 @@ function writeNextConfig() {
     }
 }
 
-async function writeOpenNextConfig(region: string) {
+async function writeOpenNextConfig(region: string, edgeFunctionPaths: EdgeFunction[]) {
+    let functions = "";
+
+    if (edgeFunctionPaths.length > 0) {
+        functions += "functions: {\n";
+        for (const index in edgeFunctionPaths) {
+            const f = edgeFunctionPaths[index];
+            functions += `   edge${index}: {
+    runtime: 'edge',
+    routes: ["${f.path}"],
```

Comment:
nit: you use just `'` everywhere else. just for consistency.

```suggestion
    routes: ['${f.path}'],
```